scripts/test-utils/matchers.ts

```diff
@@ -93,7 +88,10 @@ function toMatchFile(
 
         // If expected file doesn't exists but got a received content and if the snapshots
         // should be updated, create the new snapshot. Otherwise fails the assertion.
-        if (snapshotState._updateSnapshot === 'new' || snapshotState._updateSnapshot === 'all') {
+        if (
+            snapshotState['_updateSnapshot'] === 'new' ||
+            snapshotState['_updateSnapshot'] === 'all'
+        ) {
```

Comment:
_updateSnapshot is private in vitest which means there's probably a better way to do this, but for now this is needed to avoid annoying errors in vscode.